src/main/groovy/com/ullink/gradle/nunit/NUnitPlugin.groovy

```diff
@@ -52,7 +52,12 @@ class NUnitPlugin implements Plugin<Project> {
         if (!nunitCacheDir.exists()) {
             nunitCacheDir.mkdirs()
         }
-        def ret = new File(nunitCacheDir, NUnitName)
+        def nunitCacheDirForVersion = new File(nunitCacheDir, NUnitName)
+
+        // special handling for nunit3 flat zip file
+        def zipOutputDir = version.startsWith("3.") ? nunitCacheDirForVersion : nunitCacheDir;
```

Comment:
Well, for sure I had to change the download URL, but I cannot find that change in the commits now. Strange...